### DISCUSSION TITLE
How to disable system console message from pyodide? e.g. "Loading distutils"

### DISCUSSION THREAD

**Discussion by webdevelopland** [pyodide/pyodide#4514] [Q&A] [answered]
When you load pyodide using `loadPyodide()`, it prints to the console this:
```
Loading distutils
Loaded distutils
Python initialization complete
```
How to turn it off?

**Comment by ryanking13** [comment, marked as answer]
> Python initialization complete

This message has been removed from Pyodide 0.22.0, So you need to update Pyodide to a newer version.

> Loading distutils
Loaded distutils

This message will not be shown at `loadPyodide()` if you update to a newer version of Pyodide, too. However it will be shown when you do `pyodide.loadPackage()`. To hide this message, you can pass additional `messageCallback` parameter to loadPackage: e.g. `pyodide.loadPackage("<pkgname", { messageCallback: () => {} })`

**Reply by webdevelopland** [reply]
Thank you! That worked
